enhancements/sig-architecture/33-detached-deploy-mode/detached-deploy-mode.md

```diff
@@ -0,0 +1,217 @@
+        |  management-cluster                                     |                                               
+        |                                                         |                                               
+        |  +----------------+  +-------------+ +------------+     |                                               
+        |  |deployment:     |  | deployment: | |deployment: |     |                                               
+        |  |cluster-manager |  | registration| |placement   | etc.|                                               
+        |  |                |  |             | |            |     |                                               
+        |  +------|---------+  +------|------+ +------|-----+     |                                               
+        |         |                   |               |           |                                               
+        +---------|-------------------|---------------|-----------+                                               
+                  |                   |               |                                                           
+                  |--------------|--------------------+                                                           
+                                 |watch,create,update,delete...                                                   
+                                 |                                                                                
+        +------------------------v--------------------------------+                                               
+        |  hub-cluster                                            |       registrater       +-----------------+   
+        |                                                         <-------------------------> managed cluster |   
+        |  +-------+ +-----------+ +---------------+              |                         +-----------------+   
+        |  | CRDs  | |APIServices| |Configurations | etc.         |                                               
+        |  |       | |           | |               |              |                                               
+        |  +-------+ +-----------+ +---------------+              |                                               
+        +---------------------------------------------------------+
+```
+
+The **hub-cluster** is where hub is installed and where the cluster agent should register.
+ 
+The **management-cluster** is the cluster where deployments are actually running on. The deployments of several hub-clusters could run in one management-cluster and be divided by different namespaces.
+
+Note that cluster-manager is deploy on **management-cluster** but configed with a kubeconfig secret (in the `open-cluster-management` namespace) named ”<cluster-manager-name>-hub-kubeconfig” which is a cluster-admin kubeconfig of the **hub-cluster**.
```

Comment:
`<cluster-manager-name>-hub-kubeconfig` means for the external hub kubeconfig? Please confirm it is always deployed in the `open-cluster-management`?

Reply:
Yes, `<cluster-manager-name>-hub-kubeconfig` means the external hub kubeconfig.  And the current approach is all of cluster-manager's external-hub-kubeconfig will be depolyed in the `open-cluster-management` namespace.

Reply:
Users create `<cluster-manager-name>-hub-kubeconfig` in `open-cluster-management` ns, then the operator copy it to the installation ns?

Reply:
It could be: operator use this <cluster-manager-name>-hub-kubeconfig to generate proper SA-kubeconfig and install them in respective installation ns.

Reply:
Solved, apply secret after namespace is created